src/test/java/io/xstefank/wildfly/bot/PROpenedTest.java

```diff
@@ -298,7 +296,6 @@ void testDirectoriesMentionsNewFileInDiff() throws IOException {
                 Mockito.verify(mockedPR, Mockito.times(2)).listFiles();
                 Mockito.verify(mockedPR).comment("/cc @7125767235");
                 Mockito.verify(mockedPR, Mockito.times(1)).listComments();
-                Mockito.verifyNoMoreInteractions(mockedPR);
```

Comment:
wouldn't it be better to list individual invocations since we test them all here?